### DISCUSSION TITLE
CMake Error at ClangTargets.cmake:709 (message)  libclang-13.so.13.0.0 does not exist

### DISCUSSION THREAD

**Discussion by youngallien** [halide/Halide#6299] [Q&A] [answered]
Hi, when i build Halide with CMake on Ubuntu-18.04, it run into this error:
`CMake Error at /usr/lib/llvm-13/lib/cmake/clang/ClangTargets.cmake:709 (message):
  The imported target "libclang" references the file

     "/usr/lib/llvm-13/lib/libclang-13.so.13.0.0"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/lib/llvm-13/lib/cmake/clang/ClangTargets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/lib/llvm-13/lib/cmake/clang/ClangConfig.cmake:20 (include)
  dependencies/llvm/CMakeLists.txt:14 (find_package)
`
Any help will be appreciated!

**Comment by LebedevRI** [comment, marked as answer]
Not a halide bug.

This is a debian/ubuntu package bug https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=995483,
which has now been fixed. Feel free to close this bug.
